source/_components/media_player.pjlink.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "PJLink"
+description: "Instructions on how to integrate PJLink enabled projectors into Home Assistant."
+date: 2018-06-21 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Media Player
+ha_release: TBD
+ha_iot_class: "Local Polling"
+---
+
+The `pjlink` platform allows you to control from Home Assistant, projectors with support for the [PJLink protocol](http://pjlink.jbmia.or.jp/english/index.html).
+
+## {% linkable_title Configuration %}
+
+To add a PJLink projector  to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: pjlink
+    host: 192.168.1.2
+    port: 4352
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.